plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+		);
+		$pricing_columns         = $product_pricing_section->add_block(
+			[
+				'id'        => 'product-pricing-group-pricing-columns',
+				'blockName' => 'core/columns',
+			]
+		);
+		$pricing_column_1        = $pricing_columns->add_block(
+			[
+				'id'         => 'product-pricing-group-pricing-column-1',
+				'blockName'  => 'core/column',
+				'attributes' => [
+					'templateLock' => 'all',
+				],
+			]
+		);
+		$pricing_column_1->add_block(
+			[
+				'id'         => 'product-pricing-regular-price',
+				'blockName'  => 'woocommerce/product-regular-price-field',
+				'attributes' => [
+					'name'  => 'regular_price',
+					'label' => __( 'List price', 'woocommerce' ),
+				],
+			]
+		);
+		$pricing_column_2 = $pricing_columns->add_block(
+			[
+				'id'         => 'product-pricing-group-pricing-column-2',
+				'blockName'  => 'core/column',
+				'attributes' => [
+					'templateLock' => 'all',
+				],
+			]
+		);
+		$pricing_column_2->add_block(
+			[
+				'id'         => 'product-pricing-sale-price',
+				'blockName'  => 'woocommerce/product-sale-price-field',
+				'attributes' => [
+					'label' => __( 'Sale price', 'woocommerce' ),
+				],
+			]
+		);
+		$product_pricing_section->add_block(
+			[
+				'id'        => 'product-pricing-schedule-sale-fields',
+				'blockName' => 'woocommerce/product-schedule-sale-fields',
+			]
+		);
+		$product_pricing_section->add_block(
+			[
+				'id'         => 'product-sale-tax',
+				'blockName'  => 'woocommerce/product-radio-field',
+				'attributes' => [
+					'title'    => __( 'Charge sales tax on', 'woocommerce' ),
+					'property' => 'tax_status',
+					'options'  => [
+						[
+							'label' => __( 'Product and shipping', 'woocommerce' ),
+							'value' => 'taxable',
+						],
+						[
+							'label' => __( 'Only shipping', 'woocommerce' ),
+							'value' => 'shipping',
+						],
+						[
+							'label' => __( "Don't charge tax", 'woocommerce' ),
+							'value' => 'none',
+						],
+					],
+				],
+			]
+		);
+		$pricing_advanced_block = $product_pricing_section->add_block(
+			[
+				'id'         => 'product-pricing-advanced',
+				'blockName'  => 'woocommerce/product-collapsible',
+				'attributes' => [
+					'toggleText'       => __( 'Advanced', 'woocommerce' ),
+					'initialCollapsed' => true,
+					'persistRender'    => true,
+				],
+			]
+		);
+		$pricing_advanced_block->add_block(
+			[
+				'id'         => 'product-tax-class',
+				'blockName'  => 'woocommerce/product-radio-field',
+				'attributes' => [
+					'title'       => __( 'Tax class', 'woocommerce' ),
+					'description' => sprintf(
+					/* translators: %1$s: Learn more link opening tag. %2$s: Learn more link closing tag.*/
+						__( 'Apply a tax rate if this product qualifies for tax reduction or exemption. %1$sLearn more%2$s.', 'woocommerce' ),
+						'<a href="https://woocommerce.com/document/setting-up-taxes-in-woocommerce/#shipping-tax-class" target="_blank" rel="noreferrer">',
+						'</a>'
+					),
```

Comment:
Any reason why `sprintf` was used instead of `createInterpolateElement`?

Reply:
Same as above: https://github.com/woocommerce/woocommerce/pull/39814#discussion_r1302177468